packages/apps/transfer/src/pages/transfer/account-transactions/index.tsx

```diff
@@ -142,7 +142,7 @@ const CheckTransactions: FC = () => {
                 inputProps={{
                   id: 'account-input',
                   placeholder: t('Account'),
-                  onChange: (e) =>
+                  onChange: (e: any) =>
```

Comment:
Is there really not some `...Event` type from React?

Reply:
Something like `React.ChangeEvent<HTMLInputElement>` right?